### PR TITLE
Add Medivac beds to Corpsmen vendors for all of their points.

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
This PR allows corpsmen to buy a bundle of a bed, and a beacon for the low low price of their entire budget. 
I also added a space in some mortar code, because it was bothering me.

## Why It's Good For The Game
I always wanted to have two beds to chuck people up faster. I am absolutely willing to skimp on the other gear for that
It also gives more work to doctors shipside, which is good. While it is effectively a teleporter, and you could sooort of make a discount telepad with them, I do not think the cooldown makes it any viable for any substantial effect.  

## Changelog

:cl:
add: Added medivac beds to corpsmen vendor for their entire budget
code: Added a space to the mortar code :)
/:cl:

### CODE REVIEW

**Comment by iforgotmeword** [comment]
fucking cursed

**Comment by chizzy376** [comment]
Akimbo beds

**Review by ivanmixo — APPROVED** [review]

File: code/__DEFINES/loadout.dm [hunk 1]
```diff
@@ -119,6 +119,7 @@ GLOBAL_LIST_INIT(medic_gear_listed_products, list(
 		/obj/item/clothing/glasses/hud/health = list(CAT_MEDSUP, "Medical HUD glasses", 2, "black"),
 		/obj/item/healthanalyzer/gloves = list(CAT_MEDSUP, "Health scanner gloves", 2, "black"),
 		/obj/item/defibrillator/gloves = list(CAT_MEDSUP, "Advanced medical gloves", 5, "black"),
+		/obj/effect/vendor_bundle/stretcher = list(CAT_ESS, "Medivac Stretcher", 45, "black"),
```
Comment (QualityVan) [inline code/__DEFINES/loadout.dm:122]: Did you test this? This should probably be cat_medsup, not cat_ess.

**Review by QualityVan — CHANGES_REQUESTED** [review]

File: code/__DEFINES/loadout.dm (hunk 1 above)
Comment (lbnesquik) [reply code/__DEFINES/loadout.dm]: Changed, checked, tested, works.

**Pull request merged**